triggerbot/tree_watcher.py

```diff
@@ -57,8 +58,10 @@ def __init__(self, ldap_auth, is_triggerbot_user=lambda _: True):
         self.auth = ldap_auth
         self.lower_trigger_limit = TreeWatcher.default_retry * TreeWatcher.per_push_failures
         self.log = logging.getLogger('trigger-bot')
+        logging.basicConfig()
         self.is_triggerbot_user = is_triggerbot_user
         self.global_trigger_count = 0
+        self.treeherderapi = TreeherderApi()
```

Comment:
Something that doesn't mention treeherder, but just that this is the source for data about builds for a revision.